application-analytics-ui/src/main/resources/Analytics/WebHome.xml

```diff
@@ -27,45 +27,187 @@
   <defaultLanguage>en</defaultLanguage>
   <translation>0</translation>
   <creator>xwiki:XWiki.Admin</creator>
-  <parent>Main.WebHome</parent>
+  <parent>xwiki:Main.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
   <version>1.1</version>
   <title>Analytics</title>
```

Comment:
This page should be excluded from the licensing check, in order for users to be able to edit it.
Because of this, the display is also broken. On your instance you have the page created after application install, so the license check is not applied
![image](https://github.com/xwikisas/application-analytics/assets/22794181/29e104c7-3a48-4360-9e1c-2c99086dcc0c)